### ISSUE TITLE
[a11y] Establish baseline accessibility requirements

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5427]
We should establish baseline accessibility requirements to build and test new features (and current) against. This guidance should be added to the repository as a readme.

Resources:

[+Firefox Accessibility Requirements+](https://docs.google.com/document/d/1NaCEIshLuXzqxJbpSNFmzeUigcMpReGD1JzPknnpvlo/edit?usp=sharing)

[+WCAG Guidelines+](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.1)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-3658)
┆Reporter: Lesley Norton
